### PR TITLE
RC: Supported Cloud providers regions

### PR DESCRIPTION
**Pull request by cmilesb**
DOC-3359

### CODE REVIEW

File: content/operate/rc/supported-regions.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+categories:
+- docs
+- operate
+- rc
+description: null
+linktitle: Supported regions
+weight: 90
+---
+
+Choosing your cloud provider and region may affect latency between your application and your database, and may affect what connectivity options are available for your database.
```
Comment (andy-stark-redis) [inline content/operate/rc/supported-regions.md:13]:
```suggestion
Your choice of cloud provider and region may affect latency between your application and your database, and may affect what connectivity options are available for your database.
```

File: content/operate/rc/supported-regions.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+Redis Cloud supports databases on the following cloud providers:
+- [Amazon Web Services (AWS)](#amazon-web-services)
+- [Google Cloud](#google-cloud)
+- [Microsoft Azure](#microsoft-azure)
+
+## Amazon Web Services
+
+Redis Cloud supports databases on all of the following Amazon Web Services (AWS) regions. All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases.
```
Comment (andy-stark-redis) [inline content/operate/rc/supported-regions.md:22]: This suggestion might not be correct if a "region" is understood to mean a server more than a geographical region.
```suggestion
Redis Cloud supports databases in all of the following Amazon Web Services (AWS) regions. All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. 
```

File: content/operate/rc/supported-regions.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+| Region ID      | Location                | Redis Cloud Pro | Redis Cloud Essentials |
+|----------------|-------------------------|-----------------|------------------------|
+| `af-south-1`   | Cape Town, South Africa | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `il-central-1` | Tel Aviv, Israel        | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `me-central-1` | UAE                     | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `me-south-1`   | Bahrain                 | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+## Google Cloud
+
+Redis Cloud supports databases on all of the following Google Cloud regions. All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases.
```
Comment (andy-stark-redis) [inline content/operate/rc/supported-regions.md:76]: Same as above.

**Review by andy-stark-redis — APPROVED** [review]
Few possible minor suggestions, but language LGTM.

File: content/operate/rc/supported-regions.md (hunk 3 above)
Comment (NoamSternRedis) [inline content/operate/rc/supported-regions.md:76]: Same comment as in AWS. "All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. " is not necessary IMO

File: content/operate/rc/supported-regions.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+|----------------|-------------------------|-----------------|------------------------|
+| `af-south-1`   | Cape Town, South Africa | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `il-central-1` | Tel Aviv, Israel        | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `me-central-1` | UAE                     | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `me-south-1`   | Bahrain                 | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+## Google Cloud
+
+Redis Cloud supports databases on all of the following Google Cloud regions. All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. 
+
```
Comment (NoamSternRedis) [inline content/operate/rc/supported-regions.md:77]: @cmilesb redundant parenthesis at the end of the sentence

File: content/operate/rc/supported-regions.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+| `australia-southeast1` | Sydney, Australia    | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `australia-southeast2` | Melbourne, Australia | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+### Middle East
+
+| Region ID  | Location         | Redis Cloud Pro | Redis Cloud Essentials |
+|------------|------------------|-----------------|------------------------|
+| `me-west1` | Tel Aviv, Israel | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+## Microsoft Azure
```
Comment (NoamSternRedis) [inline content/operate/rc/supported-regions.md:129]: Can we put more clear dividers (maybe also bigger font) between the cloud providers? It looks pretty similar to the dividers between the different locations insider each provider.

File: content/operate/rc/supported-regions.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+Redis Cloud Essentials is available on the following Microsoft Azure regions:
+
+| Region ID | Location        |
+|-----------|-----------------|
+| `east-us` | Virginia, USA   |
+| `west-us` | California, USA |
+
+Redis Cloud Pro is available on Azure through Azure Cache for Redis Enterprise. See [Azure Cache for Redis pricing](https://azure.microsoft.com/en-us/pricing/details/cache/) to view the list of Azure regions that support Azure Cache for Redis Enterprise.
```
Comment (NoamSternRedis) [inline content/operate/rc/supported-regions.md:138]: not bolded enough. Maybe inside of a note?

**Review by NoamSternRedis — CHANGES_REQUESTED** [review]

File: content/operate/rc/supported-regions.md (hunk 2 above)
Comment (cmilesb) [reply content/operate/rc/supported-regions.md:22]:
```suggestion
Redis Cloud supports databases in the following Amazon Web Services (AWS) regions. 
```

File: content/operate/rc/supported-regions.md (hunk 3 above)
Comment (cmilesb) [reply content/operate/rc/supported-regions.md:76]:
```suggestion
Redis Cloud supports databases in the following Google Cloud regions. 
```

File: content/operate/rc/supported-regions.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+| `af-south-1`   | Cape Town, South Africa | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `il-central-1` | Tel Aviv, Israel        | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `me-central-1` | UAE                     | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `me-south-1`   | Bahrain                 | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+## Google Cloud
+
+Redis Cloud supports databases on all of the following Google Cloud regions. All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. 
+
+Redis Cloud Pro databases on Google Cloud support [VPC Peering]({{< relref "/operate/rc/security/vpc-peering#gcp-vpc-peering" >}}) and [Private Service Connect]({{< relref "/operate/rc/security/private-service-connect" >}})).
```
Comment (cmilesb) [inline content/operate/rc/supported-regions.md:78]:
```suggestion
Redis Cloud Pro databases on Google Cloud support [VPC Peering]({{< relref "/operate/rc/security/vpc-peering#gcp-vpc-peering" >}}) and [Private Service Connect]({{< relref "/operate/rc/security/private-service-connect" >}}).
```

File: content/operate/rc/supported-regions.md (hunk 5 above)
Comment (cmilesb) [reply content/operate/rc/supported-regions.md:129]: I made the location headings smaller, let me know what you think.

File: content/operate/rc/supported-regions.md (hunk 6 above)
Comment (cmilesb) [reply content/operate/rc/supported-regions.md:139]: I put this part inside of a note, let me know what you think.

**Review by NoamSternRedis — APPROVED** [review]

**Pull request merged**